source/OptimaJet.DWKit.StarterApplication/Utility/SystemWebClient.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+using System.Net;
+
+namespace OptimaJet.DWKit.StarterApplication.Utility
+{
+    public class SystemWebClient : WebClient, IWebClient
+    {
+
```

Comment:
So this is here just so you can scope it?

Reply:
@davidmoore1: Sorry, I didn't see the comment.  This is here so that it can be brought in as an injected dependency that can be mocked in the tests.